conf/messages.cy

```diff
@@ -2,32 +2,32 @@ service.text = Dyma'ch gwasanaeth newydd
 general.page-title = Datganiad hygyrchedd ar gyfer {0} – GOV.UK
 general.page-title.ios = Datganiad hygyrchedd ar gyfer {0} – GOV.UK
 general.page-title.android = Datganiad hygyrchedd ar gyfer {0} – GOV.UK
-general.service=service
-general.app=app
+general.service=gwasanaeth
+general.app=ap
```

Comment:
this is definitely `ap` not `app`?  it's just that there are occurrences of `app` in various messages still - is it contextual as to which is used?  or is this just because we're getting the translations piecemeal?